collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -969,4 +720,70 @@ abstract class EntitiesRepositoryTest {
         val wines = repository.query("wines", Query.NumericNotEq("score", 5.0))
         assertThat(wines, containsInAnyOrder(sameEntityAs(dows)))
     }
+
+    @Test
+    fun `#query without query returns empty list when there are not entities`() {
+        val repository = buildSubject()
+        assertThat(repository.query("wines").size, equalTo(0))
+    }
+
+    @Test
+    fun `#query without query returns entities for list`() {
+        val repository = buildSubject()
+
+        val wine = Entity.New(
+            "1",
+            "Léoville Barton 2008",
+            version = 2,
+            trunkVersion = 1
+        )
+
+        val whisky = Entity.New(
+            "2",
+            "Lagavulin 16",
+            version = 3,
+            trunkVersion = 1
+        )
+
+        repository.save("wines", wine)
+        repository.save("whiskys", whisky)
+
+        val wines = repository.query("wines")
+        assertThat(wines.size, equalTo(1))
+        assertThat(wines[0], sameEntityAs(wine))
+
+        val whiskys = repository.query("whiskys")
+        assertThat(whiskys.size, equalTo(1))
+        assertThat(whiskys[0], sameEntityAs(whisky))
+    }
+
+    @Test
+    @Ignore("https://github.com/getodk/collect/issues/6615")
```

Comment:
I think we should add a note in that issue to remember that these tests should be unblocked.